src/Tizen.MachineLearning.Inference/Tizen.MachineLearning.Inference/MlInformation.cs

```diff
@@ -0,0 +1,286 @@
+            NNStreamer.CheckNNStreamerSupport();
+
+            if (handle == IntPtr.Zero)
+                throw NNStreamerExceptionFactory.CreateException(NNStreamerError.InvalidParameter, "The information handle is null");
+
+            _handle = handle;
+            _type = type;
+            _fromList = fromList;
+        }
+
+        /// <summary>
+        /// Destroys the MlInformation resource.
+        /// </summary>
+        /// <since_tizen> 13 </since_tizen>
+        ~MlInformation() {
+            Dispose(false);
+        }
+
+        /// <summary>
+        /// Releases any unmanaged resources used by this object.
+        /// </summary>
+        /// <since_tizen> 13 </since_tizen>
+        public void Dispose()
+        {
+            Dispose(true);
+            GC.SuppressFinalize(this);
+        }
+
+        /// <summary>
+        /// Releases any unmanaged resources used by this object. Can also dispose any other disposable objects.
+        /// </summary>
+        /// <param name="disposing">If true, disposes any disposable objects. If false, does not dispose disposable objects.</param>
+        protected virtual void Dispose(bool disposing)
+        {
+            if (_disposed)
+                return;
+
+            if (disposing)
+            {
+                // release managed object
+            }
+
+            // release unmanaged objects
+            if (_handle != IntPtr.Zero)
+            {
+                NNStreamerError ret = NNStreamerError.InvalidParameter;
+
+                switch(_type)
+                {
+                    case InfoType.Option:
+                        ret = Interop.Util.DestroyOption(_handle);
+                        break;
+                    case InfoType.Information:
+                        if (!_fromList) {
+                            ret = Interop.Util.DestroyInformation(_handle);
+                        } else {
+                            /* Information handle is from MlInformationList should not be released */
+                            ret = NNStreamerError.None;
+                        }
+                        break;
+                }
+
+                NNStreamer.CheckException(ret, "Failed to destroy the information");
+
+                _handle = IntPtr.Zero;
+            }
+
+            _disposed = true;
+        }
+
+        /// <summary>
+        /// Internal method to get the native handle of MlInformation.
+        /// </summary>
+        /// <returns>The native handle</returns>
+        /// <since_tizen> 13 </since_tizen>
+        internal IntPtr GetHandle()
+        {
+            return _handle;
+        }
+
+        /// <summary>
+        /// Sets a new key-value in MlInformation instance.
+        /// </summary>
+        /// <param name="key">The key to be set.</param>
+        /// <param name="value">The value to be set.</param>
+        /// <feature>http://tizen.org/feature/machine_learning.inference</feature>
```

Comment:
In the case of ml_inofrmation_*, it will be used in ml-service (and training?) later.
How about implementing these as a common part?

Tizen.MachineLearning or Tizen.MachineLearning.Common feature

Reply:
Tizen.MachineLearning.Inference must be included to use `MlInformation.GetTensorsData`.
So how about using Tizen.MachineLearning.Inference namespace like any other common part so that user don't have to declare multiple namespaces?

Reply:
Okay, let's use the Tizen.MachineLearning.Inference namespace now. Consider separating them to common parts when it becomes necessary later.